CMakePresets.json

```diff
@@ -21,7 +21,7 @@
       "displayName": "Windows x64 Debug",
       "description": "Target Windows with the Visual Studio development environment.",
       "generator": "Ninja",
-      "binaryDir": "${sourceDir}/out/build/${presetName}",
+      "binaryDir": "${sourceDir}/out/build/Debug",
```

Comment:
better to just change the "name" above to `Debug` or `Release` below instead of hard-coding it.  I've made this change in a few of my PRs as well.